storage/replica.go

```diff
@@ -964,11 +969,9 @@ func (r *Replica) addWriteCmd(ctx context.Context, ba roachpb.BatchRequest, wg *
 		case <-ctx.Done():
 			return br, roachpb.NewError(ctx.Err())
```

Comment:
while you're here, can you either make this not a `return` or make the line above a `return`?